src/test/java/app/PopulatorTestUtil.java

```diff
@@ -153,6 +153,13 @@ public List<Review> createReviews(List<User> users, List<Game> games) {
                         games.get(1).getId(),
                         2.8,
                         "Bad game"
+                ),
+                new Review(
+                        null,
+                        users.get(2).getUsername(),
+                        games.get(1).getId(),
+                        5.0,
+                        "Mid game"
```

Comment:
What is the purpose of this? If I recall correctly, `users.get(2)` would return the admin user. Is this intended behaviour?

Reply:
The purpose was to have more users review the same game for more test data to be used in a get all reviews method, but not for admin user - must not have seen that part

Reply:
Currently two users have reviewed the same game. If a third is desired then perhaps a third user could be added to the PopulatorTestUtil createUsers